roles/foreman/tasks/main.yaml

```diff
@@ -145,6 +154,7 @@
   theforeman.foreman.smart_proxy:
     name: "{{ ansible_fqdn }}"
     url: "https://{{ ansible_fqdn }}:9090"
-    server_url: "http://{{ ansible_fqdn }}:3000"
+    server_url: "{{ foreman_url }}"
+    validate_certs: false # TODO
```

Comment:
We don't support `ca_path` today in FAM. Should we?

Reply:
I was looking for it and now I'm wondering if `validate_certs: '{{ ca_certificate }}'` works. https://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification says `verify='/path/to/certfile'` is supported and looking at https://github.com/theforeman/foreman-ansible-modules/blob/2343e66adf65aac42f2d26a076d555102320bcc6/plugins/inventory/foreman.py#L266 I'd think that works. Not sure about https://github.com/theforeman/foreman-ansible-modules/blob/2343e66adf65aac42f2d26a076d555102320bcc6/plugins/module_utils/foreman_helper.py#L617 but if we go that route, it'd make sense to me to copy requests' API because it saves a parameter.

Reply:
It saves one, yes, but diverges from `ansible.builtin.uri` API, which is the one Ansible users are more familiar with.

The problem (today) is https://github.com/theforeman/foreman-ansible-modules/blob/2343e66adf65aac42f2d26a076d555102320bcc6/plugins/module_utils/foreman_helper.py#L373 which makes it a boolean, not a string.